src/pages/todo/actions.js

```diff
@@ -1,3 +1,8 @@
 import { createRequestTypes } from '../../common/api'
 
 export const LIST_TODOS = createRequestTypes('LIST_TODOS');
```

Comment:
@vinhkq 
`export const DETAIL_TODOS = createRequestTypes('DETAIL_TODOS');`
should be
`export const VIEW_TODO = createRequestTypes('VIEW_TODO');`
as in master branch.